app/assets/stylesheets/modern-business.scss

```diff
@@ -259,6 +267,7 @@ footer {
   font-family: 'Quattrocento', serif;
   font-size: 40px;
   text-align: center;
+  color: #001B4D;
```

Comment:
Color literals like `#001B4D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#001B4D` should be written as `#001b4d`